addons/website_event/models/event_event.py

```diff
@@ -527,10 +527,18 @@ def _search_get_detail(self, website, order, options):
             mapping['description'] = {'name': 'subtitle', 'type': 'text', 'match': True}
         if with_date:
             mapping['detail'] = {'name': 'range', 'type': 'html'}
+
+        def search_in_address(env, search_term):
```

Comment:
can we add a comment to explain why we need to do it this way ? (bypass access rules on partner, but not on event)